test/e2e/common/node/pods.go

```diff
@@ -948,10 +949,11 @@ var _ = SIGDescribe("Pods", func() {
 			return false, nil
 		})
 		if err != nil {
-			p, _ := f.ClientSet.CoreV1().Pods(testNamespaceName).Get(context.TODO(), testPodName, metav1.GetOptions{})
-			framework.Logf("Pod: %+v", p)
+			framework.Logf("failed to see event that pod is created: %v", err)
 		}
-		framework.ExpectNoError(err, "failed to see Pod %v in namespace %v running", testPod.ObjectMeta.Name, testNamespaceName)
+		p, err := f.ClientSet.CoreV1().Pods(testNamespaceName).Get(context.TODO(), testPodName, metav1.GetOptions{})
+		framework.ExpectNoError(err, "failed to get Pod %v in namespace %v", testPodName, testNamespaceName)
+		framework.ExpectEqual(p.Status.Phase, v1.PodRunning, "failed to see Pod %v in namespace %v running", p.ObjectMeta.Name, testNamespaceName)
```

Comment:
This makes the test allow to get an error of the event and check the pod status.
Is the expected error a timeout here?

Reply:
Thanks for your explanation @aojea 
Ideally it is great to make events more reliable in long-term, but this is one of conformance tests and that should be stable anywhere.
This is a good solution at this time, and it is fine to investigate from this additional `err` message later.

Reply:
I think you're talking about two different types of events. Event API objects should not be used in e2e tests. Watch events on a pod watch are perfectly valid to use

Reply:
Oh, yeah, I was thinking these were v1.Event , not watch event

Reply:
The last failed result after this PR is merged shows [this message](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-conformance-kind-ga-only-parallel/1508104371345297408). And the code is [here](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/common/node/pods.go#L1067). So, the pod still remains after timeout (1*time.Minute). Then, should we just make it longer?

Reply:
did we look at what is going on in the kubelet with that pod between when deletion was requested and a minute later?

Reply:
The error message for my previous comment was as below:

```
Kubernetes e2e suite: [sig-node] Pods should run through the lifecycle of Pods and PodStatus [Conformance] expand_less	1m10s
{ Failure test/e2e/framework/framework.go:633
Mar 27 15:59:10.464: pod pod-test found in namespace pods-7010, but it should be deleted: {"metadata":{"name":"pod-test","namespace":"pods-7010","uid":"4b3659f3-ec19-4d5d-ace1-a82e77dd289b","resourceVersion":"14845","creationTimestamp":"2022-03-27T15:58:01Z","deletionTimestamp":"2022-03-27T15:58:11Z","deletionGracePeriodSeconds":1,"labels":{"test-pod":"patched","test-pod-static":"true"},"managedFields":[{"manager":"e2e.test","operation":"Update","apiVersion":"v1","time":"2022-03-27T15:58:10Z","fieldsType":"FieldsV1","fieldsV1":{"f:metadata":{"f:labels":{".":{},"f:test-pod":{},"f:test-pod-static":{}}},"f:spec":{"f:containers":{"k:{\"name\":\"pod-test\"}":{".":{},"f:image":{},"f:imagePullPolicy":{},"f:name":{},"f:resources":{},"f:terminationMessagePath":{},"f:terminationMessagePolicy":{}}},"f:dnsPolicy":{},"f:enableServiceLinks":{},"f:restartPolicy":{},"f:schedulerName":{},"f:securityContext":{},"f:terminationGracePeriodSeconds":{}}}},{"manager":"kubelet","operation":"Update","apiVersion":"v1","time":"2022-03-27T15:58:42Z","fieldsType":"FieldsV1","fieldsV1":{"f:status":{"f:conditions":{"k:{\"type\":\"ContainersReady\"}":{".":{},"f:lastProbeTime":{},"f:lastTransitionTime":{},"f:status":{},"f:type":{}},"k:{\"type\":\"Initialized\"}":{".":{},"f:lastProbeTime":{},"f:lastTransitionTime":{},"f:status":{},"f:type":{}},"k:{\"type\":\"Ready\"}":{".":{},"f:lastProbeTime":{},"f:lastTransitionTime":{},"f:status":{},"f:type":{}}},"f:containerStatuses":{},"f:hostIP":{},"f:phase":{},"f:podIP":{},"f:podIPs":{".":{},"k:{\"ip\":\"10.244.1.177\"}":{".":{},"f:ip":{}}},"f:startTime":{}}},"subresource":"status"}]},"spec":{"volumes":[{"name":"kube-api-access-skdr2","projected":{"sources":[{"serviceAccountToken":{"expirationSeconds":3607,"path":"token"}},{"configMap":{"name":"kube-root-ca.crt","items":[{"key":"ca.crt","path":"ca.crt"}]}},{"downwardAPI":{"items":[{"path":"namespace","fieldRef":{"apiVersion":"v1","fieldPath":"metadata.namespace"}}]}}],"defaultMode":420}}],"containers":[{"name":"pod-test","image":"k8s.gcr.io/e2e-test-images/httpd:2.4.38-2","resources":{},"volumeMounts":[{"name":"kube-api-access-skdr2","readOnly":true,"mountPath":"/var/run/secrets/kubernetes.io/serviceaccount"}],"terminationMessagePath":"/dev/termination-log","terminationMessagePolicy":"File","imagePullPolicy":"IfNotPresent"}],"restartPolicy":"Always","terminationGracePeriodSeconds":1,"dnsPolicy":"ClusterFirst","serviceAccountName":"default","serviceAccount":"default","nodeName":"kind-worker","securityContext":{},"schedulerName":"default-scheduler","tolerations":[{"key":"node.kubernetes.io/not-ready","operator":"Exists","effect":"NoExecute","tolerationSeconds":300},{"key":"node.kubernetes.io/unreachable","operator":"Exists","effect":"NoExecute","tolerationSeconds":300}],"priority":0,"enableServiceLinks":true,"preemptionPolicy":"PreemptLowerPriority"},"status":{"phase":"Running","conditions":[{"type":"Initialized","status":"True","lastProbeTime":null,"lastTransitionTime":"2022-03-27T15:58:01Z"},{"type":"Ready","status":"True","lastProbeTime":null,"lastTransitionTime":"2022-03-27T15:58:03Z"},{"type":"ContainersReady","status":"True","lastProbeTime":null,"lastTransitionTime":"2022-03-27T15:58:03Z"},{"type":"PodScheduled","status":"True","lastProbeTime":null,"lastTransitionTime":"2022-03-27T15:58:01Z"}],"hostIP":"172.18.0.4","podIP":"10.244.1.177","podIPs":[{"ip":"10.244.1.177"}],"startTime":"2022-03-27T15:58:01Z","containerStatuses":[{"name":"pod-test","state":{"running":{"startedAt":"2022-03-27T15:58:03Z"}},"lastState":{},"ready":true,"restartCount":0,"image":"k8s.gcr.io/e2e-test-images/agnhost:2.33","imageID":"k8s.gcr.io/e2e-test-images/agnhost@sha256:5b3a9f1c71c09c00649d8374224642ff7029ce91a721ec9132e6ed45fa73fd43","containerID":"containerd://e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9","started":true}],"qosClass":"BestEffort"}}
Expected an error to have occurred.  Got:
    <nil>: nil
test/e2e/common/node/pods.go:1067}
```

And the kubelet [log](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/108990/pull-kubernetes-conformance-kind-ga-only-parallel/1508104371345297408/artifacts/logs/kind-worker/kubelet.log) for the node (kind-worker) where the pod(4b3659f3-ec19-4d5d-ace1-a82e77dd289b) was running was as below:

The test expects that the pod wasn't found at 15:59:10, but it still seemed being processed around 15:59:20, therefore giving a bit more time would make the test pass.

```
grep 4b3659f3-ec19-4d5d-ace1-a82e77dd289b kubelet.log 
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.617458     259 pod_workers.go:571] "Pod is being synced for the first time" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.617508     259 pod_workers.go:888] "Processing pod event" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=0
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.617541     259 kubelet.go:1501] "syncPod enter" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.632957     259 factory.go:262] Factory "containerd" was unable to handle container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b"
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.632985     259 factory.go:262] Factory "systemd" was unable to handle container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b"
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.632996     259 factory.go:258] Using factory "raw" for container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b"
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.633160     259 manager.go:988] Added container: "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b" (aliases: [], namespace: "")
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.633293     259 handler.go:325] Added event &{/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b 0001-01-01 00:00:00 +0000 UTC containerCreation {<nil>}}
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.633337     259 container.go:531] Start housekeeping for container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b"
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.633488     259 handler.go:179] Unable to get Process Stats: couldn't open cpu cgroup procs file /sys/fs/cgroup/cpu,cpuacct/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/cgroup.procs : open /sys/fs/cgroup/cpu,cpuacct/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/cgroup.procs: no such file or directory
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.752128     259 reconciler.go:219] "operationExecutor.VerifyControllerAttachedVolume started for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.853768     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.853806     259 reconciler.go:256] "operationExecutor.MountVolume started for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.853888     259 projected.go:183] Setting up volume kube-api-access-skdr2 for pod 4b3659f3-ec19-4d5d-ace1-a82e77dd289b at /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:01 kind-worker kubelet[259]: I0327 15:58:01.954737     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.056541     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.160902     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.261871     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.362884     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.397818     259 empty_dir_linux.go:99] Statfs_t of /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2: {Type:61267 Bsize:4096 Blocks:193195126 Bfree:185169847 Bavail:175338654 Files:49135616 Ffree:48736261 Fsid:{Val:[-173936833 -1868145044]} Namelen:255 Frsize:4096 Flags:4128 Spare:[0 0 0 0]}
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.397862     259 empty_dir.go:334] pod 4b3659f3-ec19-4d5d-ace1-a82e77dd289b: mounting tmpfs for volume wrapped_kube-api-access-skdr2
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.397879     259 mount_linux.go:183] Mounting cmd (mount) with arguments (-t tmpfs tmpfs /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2)
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.401339     259 atomic_writer.go:181] pod pods-7010/pod-test volume kube-api-access-skdr2: performed write of new data to ts data directory: /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2/..2022_03_27_15_58_02.2927280351
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.401452     259 operation_generator.go:712] "MountVolume.SetUp succeeded for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.819381     259 factory.go:258] Using factory "containerd" for container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64"
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.865470     259 manager.go:988] Added container: "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64" (aliases: [2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64 /kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64], namespace: "containerd")
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.865812     259 handler.go:325] Added event &{/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64 2022-03-27 15:58:02.815492011 +0000 UTC containerCreation {<nil>}}
Mar 27 15:58:02 kind-worker kubelet[259]: I0327 15:58:02.865869     259 container.go:531] Start housekeeping for container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64"
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.106093     259 factory.go:258] Using factory "containerd" for container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9"
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.197978     259 manager.go:988] Added container: "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9" (aliases: [e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9 /kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9], namespace: "containerd")
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.198257     259 handler.go:325] Added event &{/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9 2022-03-27 15:58:03.103517434 +0000 UTC containerCreation {<nil>}}
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.198400     259 container.go:531] Start housekeeping for container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9"
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.215210     259 kubelet.go:1503] "syncPod exit" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b isTerminal=false
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.215239     259 pod_workers.go:988] "Processing pod event done" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=0
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.345010     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9" oldState=non-existent newState=running
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.345023     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64" oldState=non-existent newState=running
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.349691     259 kubelet.go:2098] "SyncLoop (PLEG): event for pod" pod="pods-7010/pod-test" event=&{ID:4b3659f3-ec19-4d5d-ace1-a82e77dd289b Type:ContainerStarted Data:e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9}
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.349721     259 kubelet.go:2098] "SyncLoop (PLEG): event for pod" pod="pods-7010/pod-test" event=&{ID:4b3659f3-ec19-4d5d-ace1-a82e77dd289b Type:ContainerStarted Data:2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64}
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.349740     259 pod_workers.go:888] "Processing pod event" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=0
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.349756     259 kubelet.go:1501] "syncPod enter" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.350509     259 kubelet.go:1503] "syncPod exit" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b isTerminal=false
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.350536     259 pod_workers.go:988] "Processing pod event done" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=0
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.350550     259 pod_workers.go:888] "Processing pod event" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=0
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.478782     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") Volume is already mounted to pod, but remount was requested." pod="pods-7010/pod-test"
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.479090     259 projected.go:183] Setting up volume kube-api-access-skdr2 for pod 4b3659f3-ec19-4d5d-ace1-a82e77dd289b at /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.479437     259 atomic_writer.go:161] pod pods-7010/pod-test volume kube-api-access-skdr2: no update required for target directory /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:03 kind-worker kubelet[259]: I0327 15:58:03.479482     259 operation_generator.go:712] "MountVolume.SetUp succeeded for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:04 kind-worker kubelet[259]: I0327 15:58:04.367082     259 kubelet.go:1501] "syncPod enter" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:04 kind-worker kubelet[259]: I0327 15:58:04.367689     259 kubelet.go:1503] "syncPod exit" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b isTerminal=false
Mar 27 15:58:04 kind-worker kubelet[259]: I0327 15:58:04.367720     259 pod_workers.go:988] "Processing pod event done" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=0
Mar 27 15:58:04 kind-worker kubelet[259]: I0327 15:58:04.491947     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") Volume is already mounted to pod, but remount was requested." pod="pods-7010/pod-test"
Mar 27 15:58:04 kind-worker kubelet[259]: I0327 15:58:04.493249     259 projected.go:183] Setting up volume kube-api-access-skdr2 for pod 4b3659f3-ec19-4d5d-ace1-a82e77dd289b at /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:04 kind-worker kubelet[259]: I0327 15:58:04.493426     259 atomic_writer.go:161] pod pods-7010/pod-test volume kube-api-access-skdr2: no update required for target directory /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:04 kind-worker kubelet[259]: I0327 15:58:04.493466     259 operation_generator.go:712] "MountVolume.SetUp succeeded for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.394170     259 status_manager.go:685] "Patch status for pod" pod="pods-7010/pod-test" patch="{\"metadata\":{\"uid\":\"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\"},\"status\":{\"$setElementOrder/conditions\":[{\"type\":\"Initialized\"},{\"type\":\"Ready\"},{\"type\":\"ContainersReady\"},{\"type\":\"PodScheduled\"}],\"conditions\":[{\"lastProbeTime\":null,\"lastTransitionTime\":\"2022-03-27T15:58:01Z\",\"status\":\"True\",\"type\":\"Initialized\"},{\"lastProbeTime\":null,\"lastTransitionTime\":\"2022-03-27T15:58:03Z\",\"status\":\"True\",\"type\":\"Ready\"},{\"lastProbeTime\":null,\"lastTransitionTime\":\"2022-03-27T15:58:03Z\",\"status\":\"True\",\"type\":\"ContainersReady\"}],\"containerStatuses\":[{\"containerID\":\"containerd://e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9\",\"image\":\"k8s.gcr.io/e2e-test-images/agnhost:2.33\",\"imageID\":\"k8s.gcr.io/e2e-test-images/agnhost@sha256:5b3a9f1c71c09c00649d8374224642ff7029ce91a721ec9132e6ed45fa73fd43\",\"lastState\":{},\"name\":\"pod-test\",\"ready\":true,\"restartCount\":0,\"started\":true,\"state\":{\"running\":{\"startedAt\":\"2022-03-27T15:58:03Z\"}}}],\"hostIP\":\"172.18.0.4\",\"phase\":\"Running\",\"podIP\":\"10.244.1.177\",\"podIPs\":[{\"ip\":\"10.244.1.177\"}],\"startTime\":\"2022-03-27T15:58:01Z\"}}"
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.419915     259 pod_workers.go:888] "Processing pod event" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=0
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.419945     259 kubelet.go:1501] "syncPod enter" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.420708     259 kuberuntime_container.go:722] "Killing container with a grace period" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerName="pod-test" containerID="containerd://e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9" gracePeriod=2
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.460679     259 pod_workers.go:625] "Pod is marked for graceful deletion, begin teardown" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.460699     259 pod_workers.go:761] "Cancelling current pod sync" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=1
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.511489     259 kuberuntime_container.go:731] "Container exited normally" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerName="pod-test" containerID="containerd://e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9"
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.519878     259 manager.go:1044] Destroyed container: "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9" (aliases: [e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9 /kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9], namespace: "containerd")
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.519894     259 handler.go:325] Added event &{/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9 2022-03-27 15:58:10.519890211 +0000 UTC m=+311.543544568 containerDeletion {<nil>}}
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.577118     259 reconciler.go:245] "Starting operationExecutor.MountVolume for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") Volume is already mounted to pod, but remount was requested." pod="pods-7010/pod-test"
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.577330     259 projected.go:183] Setting up volume kube-api-access-skdr2 for pod 4b3659f3-ec19-4d5d-ace1-a82e77dd289b at /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.577595     259 atomic_writer.go:161] pod pods-7010/pod-test volume kube-api-access-skdr2: no update required for target directory /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.577632     259 operation_generator.go:712] "MountVolume.SetUp succeeded for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"pod-test\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") " pod="pods-7010/pod-test"
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.714225     259 factory.go:258] Using factory "containerd" for container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9"
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.822060     259 manager.go:988] Added container: "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9" (aliases: [d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9 /kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9], namespace: "containerd")
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.822374     259 handler.go:325] Added event &{/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9 2022-03-27 15:58:10.712189033 +0000 UTC containerCreation {<nil>}}
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.822518     259 container.go:531] Start housekeeping for container "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9"
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.855782     259 kubelet.go:1503] "syncPod exit" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b isTerminal=false
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.855833     259 pod_workers.go:988] "Processing pod event done" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=0
Mar 27 15:58:10 kind-worker kubelet[259]: I0327 15:58:10.855846     259 pod_workers.go:888] "Processing pod event" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=1
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.410985     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9" oldState=running newState=exited
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.410998     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9" oldState=non-existent newState=running
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.426337     259 generic.go:296] "Generic (PLEG): container finished" podID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9" exitCode=2
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.426354     259 pod_workers.go:1005] "Pod worker has observed request to terminate" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.426366     259 kubelet.go:2098] "SyncLoop (PLEG): event for pod" pod="pods-7010/pod-test" event=&{ID:4b3659f3-ec19-4d5d-ace1-a82e77dd289b Type:ContainerDied Data:e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9}
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.426368     259 kubelet.go:1740] "syncTerminatingPod enter" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.426392     259 kubelet.go:2098] "SyncLoop (PLEG): event for pod" pod="pods-7010/pod-test" event=&{ID:4b3659f3-ec19-4d5d-ace1-a82e77dd289b Type:ContainerStarted Data:d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9}
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.426458     259 kubelet.go:1770] "Pod terminating with grace period" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b gracePeriod=1
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.426506     259 kuberuntime_container.go:718] "Killing container with a grace period override" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerName="pod-test" containerID="containerd://d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9" gracePeriod=1
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.426520     259 kuberuntime_container.go:722] "Killing container with a grace period" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerName="pod-test" containerID="containerd://d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9" gracePeriod=1
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.484317     259 manager.go:1044] Destroyed container: "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9" (aliases: [d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9 /kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9], namespace: "containerd")
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.484363     259 handler.go:325] Added event &{/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9 2022-03-27 15:58:11.484348146 +0000 UTC m=+312.508002497 containerDeletion {<nil>}}
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.528328     259 kuberuntime_container.go:731] "Container exited normally" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerName="pod-test" containerID="containerd://d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9"
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.571259     259 manager.go:1044] Destroyed container: "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64" (aliases: [2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64 /kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64], namespace: "containerd")
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.571298     259 handler.go:325] Added event &{/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b/2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64 2022-03-27 15:58:11.571289962 +0000 UTC m=+312.594944319 containerDeletion {<nil>}}
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.630895     259 kubelet.go:1812] "Post-termination container state" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containers=[{Name:pod-test State:exited ExitCode:0 FinishedAt:2022-03-27T15:58:11.463734023Z} {Name:pod-test State:exited ExitCode:2 FinishedAt:2022-03-27T15:58:10.43338202Z}]
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.630922     259 kubelet.go:1819] "Pod termination stopped all running containers" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.630936     259 kubelet.go:1821] "syncTerminatingPod exit" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.630946     259 pod_workers.go:1050] "Pod terminated all containers successfully" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.630965     259 pod_workers.go:988] "Processing pod event done" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=1
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.630973     259 pod_workers.go:888] "Processing pod event" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=2
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.720617     259 desired_state_of_world_populator.go:254] "Removing volume from desired state" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b volumeName="kube-api-access-skdr2"
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.797073     259 reconciler.go:190] "operationExecutor.UnmountVolume started for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") pod \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\" (UID: \"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\") "
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.797172     259 subpath_linux.go:244] Cleaning up subpath mounts for /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volume-subpaths/kube-api-access-skdr2
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.797212     259 projected.go:366] Tearing down volume kube-api-access-skdr2 for pod 4b3659f3-ec19-4d5d-ace1-a82e77dd289b at /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.797370     259 empty_dir_linux.go:99] Statfs_t of /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2: {Type:16914836 Bsize:4096 Blocks:6684886 Bfree:6684883 Bavail:6684883 Files:6684886 Ffree:6684877 Fsid:{Val:[0 0]} Namelen:255 Frsize:4096 Flags:4128 Spare:[0 0 0 0]}
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.797409     259 mount_linux.go:294] Unmounting /var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes/kubernetes.io~projected/kube-api-access-skdr2
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.799735     259 operation_generator.go:865] UnmountVolume.TearDown succeeded for volume "kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2" (OuterVolumeSpecName: "kube-api-access-skdr2") pod "4b3659f3-ec19-4d5d-ace1-a82e77dd289b" (UID: "4b3659f3-ec19-4d5d-ace1-a82e77dd289b"). InnerVolumeSpecName "kube-api-access-skdr2". PluginName "kubernetes.io/projected", VolumeGidValue ""
Mar 27 15:58:11 kind-worker kubelet[259]: I0327 15:58:11.898674     259 reconciler.go:297] "Volume detached for volume \"kube-api-access-skdr2\" (UniqueName: \"kubernetes.io/projected/4b3659f3-ec19-4d5d-ace1-a82e77dd289b-kube-api-access-skdr2\") on node \"kind-worker\" DevicePath \"\""
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.173907     259 kubelet_pods.go:1975] "Orphaned pod found, removing pod cgroups" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.187512     259 manager.go:1044] Destroyed container: "/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b" (aliases: [], namespace: "")
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.190218     259 handler.go:325] Added event &{/kubelet/kubepods/besteffort/pod4b3659f3-ec19-4d5d-ace1-a82e77dd289b 2022-03-27 15:58:12.190208845 +0000 UTC m=+313.213863236 containerDeletion {<nil>}}
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.429406     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9" oldState=running newState=exited
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.429418     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64" oldState=running newState=exited
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.436329     259 kubelet.go:1829] "syncTerminatedPod enter" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.436499     259 kubelet.go:1842] "Pod termination unmounted volumes" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.436769     259 kubelet.go:2098] "SyncLoop (PLEG): event for pod" pod="pods-7010/pod-test" event=&{ID:4b3659f3-ec19-4d5d-ace1-a82e77dd289b Type:ContainerDied Data:d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9}
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.436864     259 kubelet.go:1863] "Pod termination removed cgroups" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.436952     259 kubelet.go:1868] "Pod is terminated and will need no more status updates" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.436980     259 kubelet.go:1870] "syncTerminatedPod exit" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.436994     259 pod_workers.go:1105] "Pod is complete and the worker can now stop" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.437015     259 pod_workers.go:959] "Processing pod event done" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b updateType=2
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.436341     259 generic.go:296] "Generic (PLEG): container finished" podID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9" exitCode=0
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.437099     259 kubelet.go:2098] "SyncLoop (PLEG): event for pod" pod="pods-7010/pod-test" event=&{ID:4b3659f3-ec19-4d5d-ace1-a82e77dd289b Type:ContainerDied Data:2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64}
Mar 27 15:58:12 kind-worker kubelet[259]: I0327 15:58:12.437128     259 pod_workers.go:611] "Pod is finished processing, no further updates" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:13 kind-worker kubelet[259]: I0327 15:58:13.444335     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9" oldState=exited newState=non-existent
Mar 27 15:58:13 kind-worker kubelet[259]: I0327 15:58:13.444349     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9" oldState=exited newState=non-existent
Mar 27 15:58:28 kind-worker kubelet[259]: I0327 15:58:28.125965     259 generic.go:155] "GenericPLEG" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b containerID="2c8b4d885bc89c36714e25e7135022632c8a19dc0094969698b93f211d475d64" oldState=exited newState=non-existent
Mar 27 15:58:28 kind-worker kubelet[259]: I0327 15:58:28.150046     259 generic.go:399] "PLEG: Delete status for pod" podUID="4b3659f3-ec19-4d5d-ace1-a82e77dd289b"
Mar 27 15:58:32 kind-worker kubelet[259]: I0327 15:58:32.626864     259 kuberuntime_gc.go:343] "Removing pod logs" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:58:42 kind-worker kubelet[259]: I0327 15:58:42.990041     259 status_manager.go:685] "Patch status for pod" pod="pods-7010/pod-test" patch="{\"metadata\":{\"uid\":\"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\"},\"status\":{\"$setElementOrder/conditions\":[{\"type\":\"Initialized\"},{\"type\":\"Ready\"},{\"type\":\"ContainersReady\"},{\"type\":\"PodScheduled\"}],\"conditions\":[{\"status\":\"True\",\"type\":\"Ready\"},{\"status\":\"True\",\"type\":\"ContainersReady\"}]}}"
Mar 27 15:59:20 kind-worker kubelet[259]: I0327 15:59:20.788748     259 status_manager.go:685] "Patch status for pod" pod="pods-7010/pod-test" patch="{\"metadata\":{\"uid\":\"4b3659f3-ec19-4d5d-ace1-a82e77dd289b\"},\"status\":{\"$setElementOrder/conditions\":[{\"type\":\"Initialized\"},{\"type\":\"Ready\"},{\"type\":\"ContainersReady\"},{\"type\":\"PodScheduled\"}],\"conditions\":[{\"lastTransitionTime\":\"2022-03-27T15:58:12Z\",\"message\":\"containers with unready status: [pod-test]\",\"reason\":\"ContainersNotReady\",\"status\":\"False\",\"type\":\"Ready\"},{\"lastTransitionTime\":\"2022-03-27T15:58:12Z\",\"message\":\"containers with unready status: [pod-test]\",\"reason\":\"ContainersNotReady\",\"status\":\"False\",\"type\":\"ContainersReady\"}],\"containerStatuses\":[{\"containerID\":\"containerd://d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9\",\"image\":\"k8s.gcr.io/e2e-test-images/httpd:2.4.38-2\",\"imageID\":\"k8s.gcr.io/e2e-test-images/httpd@sha256:1b9d1b2f36cb2dbee1960e82a9344aeb11bd4c4c03abf5e1853e0559c23855e3\",\"lastState\":{\"terminated\":{\"containerID\":\"containerd://e6299e5cc1794540910b079dc908cfe698a22f7b89eda449b525e07c927e98f9\",\"exitCode\":2,\"finishedAt\":\"2022-03-27T15:58:10Z\",\"reason\":\"Error\",\"startedAt\":\"2022-03-27T15:58:03Z\"}},\"name\":\"pod-test\",\"ready\":false,\"restartCount\":1,\"started\":false,\"state\":{\"terminated\":{\"containerID\":\"containerd://d588c160d5f4b45f0c5d58718551fddef331dc078410e3b8e71ce0946db9cee9\",\"exitCode\":0,\"finishedAt\":\"2022-03-27T15:58:11Z\",\"reason\":\"Completed\",\"startedAt\":\"2022-03-27T15:58:10Z\"}}}]}}"
Mar 27 15:59:20 kind-worker kubelet[259]: I0327 15:59:20.990417     259 pod_workers.go:611] "Pod is finished processing, no further updates" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:59:21 kind-worker kubelet[259]: I0327 15:59:21.001158     259 kubelet.go:1915] "Pod has been deleted and must be killed" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:59:21 kind-worker kubelet[259]: I0327 15:59:21.001190     259 pod_workers.go:611] "Pod is finished processing, no further updates" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:59:22 kind-worker kubelet[259]: I0327 15:59:22.171537     259 pod_workers.go:1258] "Pod has been terminated and is no longer known to the kubelet, remove all history" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
Mar 27 15:59:22 kind-worker kubelet[259]: I0327 15:59:22.181724     259 kubelet_volumes.go:160] "Cleaned up orphaned pod volumes dir" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b path="/var/lib/kubelet/pods/4b3659f3-ec19-4d5d-ace1-a82e77dd289b/volumes"
Mar 27 15:59:22 kind-worker kubelet[259]: I0327 15:59:22.181970     259 kubelet_volumes.go:236] "Orphaned pod found, removing" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b
```

Reply:
- in the above last comment, does the `1m` timer starts from this log msg:
 `15:58:10.460679     259 pod_workers.go:625] "Pod is marked for graceful deletion, begin teardown" pod="pods-7010/pod-test" podUID=4b3659f3-ec19-4d5d-ace1-a82e77dd289b` ?

- the [recent logs](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/115331/pull-kubernetes-conformance-kind-ga-only-parallel/1636008441128423424/build-log.txt) also show the following error: `Mar 15 14:42:27.872: INFO: failed to see DELETED event: timed out waiting for the condition [FAILED] in [It] - test/e2e/common/node/pods.go:1070`

- this test is still flaky and I could help with increasing the timeout period as suggested in https://github.com/kubernetes/kubernetes/pull/108892#discussion_r836461132 

- flaky behaviour tracked in: https://github.com/kubernetes/kubernetes/issues/108891

Reply:
> It relies on events which are not reliable as discussed https://github.com/kubernetes/kubernetes/issues/71646.

we should not have tests that depend on an Event